app/models/lockstep/service_fabric_payment.rb

```diff
@@ -3,5 +3,34 @@ class Lockstep::ServiceFabricPayment < Lockstep::ApiRecord
   self.id_ref = "end_to_end_identifier"
   self.query_path=""
   load_schema(Schema::SfPaymentRequest)
+  
+  # Returning alwas false becase if id_ref is present in
+  # request body it returns true but in this model id_ref
+  # is required field in request body
+  # TODO: Make it generic 
+  def persisted?
+    false
+  end
```

Comment:
can you explain why we are using persisted? always as false? so even for an existing object .persisted? will give false which is wrong...

Reply:
Please suggest if there is better way

Reply:
can you please suggest how would it perform for an existing object?